ovs_container/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:centos7
+
+ENV MLNX_OFED_RHEL7_LIBS "https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.7/x86_64/UPSTREAM_LIBS/"
+ENV MLNX_OFED_VERSION "50100.0"
+
+RUN yum install -y  python3-libs unbound-libs perl pciutils libnl3 numactl\
+    python python-six iproute hostname compat-openssl10 unbound\
+    ${MLNX_OFED_RHEL7_LIBS}rdma-core-50mlnx1-1.${MLNX_OFED_VERSION}.x86_64.rpm\
+    ${MLNX_OFED_RHEL7_LIBS}libibverbs-50mlnx1-1.${MLNX_OFED_VERSION}.x86_64.rpm\
+    ${MLNX_OFED_RHEL7_LIBS}mlnx-dpdk-19.11.0-1.${MLNX_OFED_VERSION}.x86_64.rpm\
+    ${MLNX_OFED_RHEL7_LIBS}openvswitch-2.12.1-1.${MLNX_OFED_VERSION}.x86_64.rpm
+
```

Comment:
please use entrypoint so that ovs will run by when you create the pod
see [1] [2] 
[1] - https://github.com/moshe010/sriov-cni/blob/master/Dockerfile
[2] - https://github.com/moshe010/sriov-cni/blob/master/images/entrypoint.sh

Reply:
Done